addons/hr_contract/hr_contract.py

```diff
@@ -63,8 +63,14 @@ def _contracts_count(self, cr, uid, ids, field_name, arg, context=None):
 class hr_contract_type(osv.osv):
     _name = 'hr.contract.type'
     _description = 'Contract Type'
+    _order = 'sequence'
```

Comment:
order by sequence and id, so that if two sequence are identical, we keep a total order